src/components/ifxRange/ifxRange.stories.tsx

```diff
@@ -0,0 +1,85 @@
+import { action } from '@storybook/addon-actions';
+
+export default {
+  title: 'Components/Range',
+  component: 'ifx-range',
+  argTypes: {
+    value: { control: 'number' },
+    min: { control: 'number' },
+    max: { control: 'number' },
+    showPercentage: { control: 'boolean' },
+    disabled: { control: 'boolean' },
+    leftIcon: { control: 'text' },
+    rightIcon: { control: 'text' },
+    leftText: { control: 'text' },
+    rightText: { control: 'text' },
+  },
```

Comment:
There are controls for every variant along with the variant itself. There should be either a variant
or a control, not both.

Reply:
what do you mean?